PatientRegistrationApp/src/components/EligibilityQuestions.vue

```diff
@@ -0,0 +1,201 @@
+<template>
+  <v-container fill-height fluid>
+    <v-row align="center" justify="start">
+      <v-col cols="12">
+        <div>Eligibility Questions: Information collected on this form will be used to document authorization for receipt of vaccines.  
+          Your responses will be verified when receiving the vaccine.  The information will be shared through the 
+          Wisconsin Immunization Registry (WIR) with other health care providers directly involved with the 
+          patient to assure completion of the vaccine schedule.  Information collected on this form is confidential.
+        </div>
+        <div><br></div>
+      </v-col>
+    </v-row>
+    <v-row no-gutters>
+      <v-divider></v-divider>
+    </v-row>
+    <v-row align="center" justify="center">
+        <!-- County  -->
```

Comment:
Add a question for this since the following questions start at number 2? I think I would prefer this input field to be left justified, but would definitely wait for feedback from the team